plugins/experimental/collapsed_connection/MurmurHash3.cc

```diff
@@ -114,8 +114,12 @@ MurmurHash3_x86_32(const void *key, int len, uint32_t seed, void *out)
   switch (len & 3) {
   case 3:
     k1 ^= tail[2] << 16;
+  // fallthrough
```

Comment:
Kinda makes you wish there was a pragma or something, to turn off this warning for the entire switch statement, instead of all the // fallthrough comments?